src/Playwright/Transport/Channels/PageChannel.cs

```diff
@@ -148,7 +148,7 @@ internal Task SetDefaultTimeoutNoReplyAsync(float timeout)
                     ["timeout"] = timeout,
```

Comment:
According to protocol.yml timeout is no optional number, won't it throw here if the timeout is null?

Reply:
Yeah, that's a good point. Thanks for pointing that out. 

I've changed the implementation so that only the backing field can be null in which case the Property itself defaults to `PlaywrightImpl.DefaultTimeout`.